doubly_linked_list/doubly_linked_list.py

```diff
@@ -38,27 +38,55 @@ class DoublyLinkedList:
   def __init__(self, node=None):
     self.head = node
     self.tail = node
+    self.max = 0
 
   def add_to_head(self, value):
-    pass
+    if value > self.max:
+      self.max = value
+    new_link = ListNode(value)
+    if self.head == None:
```

Comment:
is self.head doesn't exist, that means the LL/DLL has o items. what does that say about the tail and the head? you're only setting a head here..